lms/templates/email/mention/subject.jinja2

```diff
@@ -1 +1 @@
-Someone has mentioned you on an annotation
\ No newline at end of file
+New activity in {{ assignment_title }}
```

Comment:
I assume `assignment_title` was already available in the context of this template

Reply:
Yes we pass the same variables to both the subject and body templates :+1: